Packs/Mattermost/Integrations/MattermostV2/MattermostV2.py

```diff
@@ -1543,13 +1572,22 @@ def send_notification(client: HTTPClient, **args):
     if entry_object:
         investigation_id = entry_object.get('investigationId')  # From server, available from demisto v6.1 and above
 
-    if message_type and message_type != MIRROR_TYPE:
+    if message_type and (message_type not in PERMITTED_NOTIFICATION_TYPES) and message_type != MIRROR_TYPE:
```

Comment:
let's talk about this tomorrow not sure I understand

Reply:
change to or on the second and

Reply:
upon further thinking, it should stay the same, as we do not want to mirror message not in the permitted notifications, and not from the mirrorType message type